docs/02-app/01-building-your-application/09-upgrading/01-codemods.mdx

```diff
@@ -24,6 +24,30 @@ Replacing `<transform>` and `<path>` with appropriate values.
 
 ## Next.js Codemods
 
+### 14
+
+#### Migrating from `next/server` to `next/og`
+
+##### `next-og-import`
+
+```bash filename="Terminal"
+npx @next/codemod@latest next-og-import .
+```
+
+This codemod moves transforms imports from `next/server` to `next/og` for usage of [Dynamic OG Image Generation](/docs/app/building-your-application/optimizing/metadata#dynamic-image-generation).
```

Comment:
Should "transforms" be pluralised here?
```suggestion
This codemod moves transform imports from `next/server` to `next/og` for usage of [Dynamic OG Image Generation](/docs/app/building-your-application/optimizing/metadata#dynamic-image-generation).
```

Reply:
Id go as far to say that we could likely remove the word "transforms" from this sentence.

Reply:
It's correct, just some specific terminology around code transformations, or transforms 🙏